notebook/converted/imputation_viz.py

```diff
@@ -123,3 +123,88 @@
 plot += ggtitle('The relationship between sample count and val set loss')
 print(plot)
 
+
+# ## Evaluate Transfering Models
+
+# In[12]:
+
+
+in_files = glob.glob('../../results/transfer.*')
+print(in_files[:5])
+
+
+# In[17]:
+
+
+metrics = pd.DataFrame()
+for path in in_files:
+    new_df = pd.read_csv(path, sep='\t')
+    
+    model_info = path.strip('.tsv').split('results/impute.')[-1]
+    model_info = model_info.split('.')        
+
+    supervised_model = model_info[4]
+    seed = model_info[6]
+    disease = model_info[3]
+             
+    new_df['supervised'] = supervised_model
+    new_df['seed'] = seed
+    new_df['disease'] = disease
+        
+    metrics = pd.concat([metrics, new_df])
+    
+metrics = metrics.reset_index()
+metrics['trial'] = metrics.index // 20
+metrics
+
+
+# In[18]:
+
+
+sepsis_df = metrics[metrics['disease'] == 'sepsis']
+tb_df = metrics[metrics['disease'] == 'tb']
+
+
+# In[19]:
+
+
+ggplot(sepsis_df, aes(x='train sample count', y='balanced_accuracy', color='factor(trial)')) + geom_line()
```

Comment:
What is each factor here? Are they different imputation proportions, or different holdout studies/holdout sets?

Also, do you have any idea what's causing the oscillations here (accuracy goes up and down as train sample count increases) in some of the trials?

Reply:
I called the datapoints corresponding to a series of models trained on increasing subsets of a single dataset split a "trial" for the sake of this notebook. I'm not really sure what causes the oscillations in accuracy, I'd guess that it's just noise caused by different training data